### ISSUE TITLE
Add the ability to render Rmds and other script collections

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcapcustodian#83]
We should add the ability to render Rmds and other script collections. That would require these steps:

- [ ] Port [`render_report.R`](https://github.com/ctsit/stp/blob/master/report/render_report.R) from STP to REDCap Custodian. 
- [ ] Change the image referenced in the Dockerfile from `rocker/tidyverse` to `rocker/verse` to get knitr support.

**Comment by ljwoodley** [comment]
Is the goal for this to be function or an example script in `/report`?

**Comment by pbchase** [comment]
> Is the goal for this to be function or an example script in `/report`?

Remind me why we didn't put `render_report.R` at `./report/render_report.R'? With the layering feature of containers, I think it would have worked fine. I know I agreed that putting it in the template was a good idea, but having deployed this in a project today, I was wishing it was at `./report/render_report.R`. Instead, I had to copy it to from the redcapcustodian template to ./report in the project where I was working, release and deploy that. I used the exact script from redcapcustodian. It felt like a pointless operation.

What's more, we did move it to `./report/render_report.R` and I wanted to revise that "standard" version, then I could copy the file from redcapcustodian to my own project dropping it in exactly the same location, then revise it to meet the needs of that project.
